swarm/metrics/flags.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package metrics
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"github.com/ethereum/go-ethereum/log"
+	gethmetrics "github.com/ethereum/go-ethereum/metrics"
+	metrics "github.com/ethersphere/go-metrics"
+	influxdb "github.com/ethersphere/go-metrics-influxdb"
+	"gopkg.in/urfave/cli.v1"
+)
+
+var (
+	metricsEndpointFlag = cli.StringFlag{
+		Name:  "metricsendpoint",
+		Usage: "metrics backend endpoint",
+		Value: "http://127.0.0.1:8086",
+	}
+	metricsDatabaseFlag = cli.StringFlag{
+		Name:  "metricsdatabase",
```

Comment:
I think this should be called `backend` rather than `database`.

Reply:
It is actually the `database` within the InfluxDB backend. If we want to support multiple backends then these flags might need to change. AFAIK we don't need multiple backends at the moment.

Reply:
Ok, so perhaps it would be better as `--metrics.influxdb.database`? I much prefer self documenting flags. At the very least, put this information in the flag description.

Reply:
Fixed.